src/App.tsx

```diff
@@ -38,58 +39,128 @@ export function getReorderedGoods(
   // eslint-disable-next-line no-console
   console.log(sortType, isReversed);
 
+  visibleGoods.sort((first, second) => {
+    switch (sortType) {
+      case SortType.ALPHABET:
+        return first.localeCompare(second);
+
+      case SortType.LENGTH:
+        return first.length - second.length;
+
+      default:
+        return 0;
+    }
+  });
```

Comment:
Don't start sorting if you don't have sort type + don't check the sort type inside the iterator, check it before

```suggestion
  switch (sortType) {
    case SortType.ALPHABET: 
      visibleGoods.sort((firstGood, secondGood) => (
        firstGood.localeCompare(secondGood);
      )
    
    case .....
```